builders/tools/pip.py

```diff
@@ -28,7 +27,7 @@ def install_pip(dir_name,version=None):
             tmp_dir = tempfile.mkdtemp()
             path = os.path.join(tmp_dir,name)
             wget('https://bootstrap.pypa.io/get-pip.py',path)
-            if subprocess.call([os.path.join(dir_name,'bin','python'),path,'--quiet']):
+            if subprocess.call([os.path.join(dir_name,'bin','python'),path]):

```

Comment:
Good question. I don't remember. I guess I wanted to see where things failed while building them and removed it for that (and then never added it back). Feel free to add it back in.